Scripts/ResetVmPermissions/FixRoleAssignments.ps1

```diff
@@ -0,0 +1,94 @@
+﻿# Example ./FixRoleAssignments.ps1 -SubscriptionIds ('sub1','sub2','sub3') -LabRoleToAdd "DevTest Labs User"
+param
+(
+    [Parameter(Mandatory=$false, HelpMessage="If you want to optionally target specific subscriptions instead of all of them, pass them in as a set")]
+    [Array] $SubscriptionIds,
+
+    [Parameter(Mandatory=$false, HelpMessage="If we should ALSO give VM Owners rights to the DevTest Lab, specify the Lab role - either DevTest Lab User, or Owner")]
+    [ValidateSet("DevTest Labs User", "Owner", "")]
+    [string] $LabRoleToAdd = ""
+)
+
+if ($SubscriptionIds -eq $null) {
+    $SubscriptionIds  = Get-AzureRmSubscription

```

Comment:
Qualify the variable to ensure it is always constructed as an array, even when only 1 record is returned.
```powershell
[Array] $SubscriptionIds = Get-AzureRmSubscription
```